readme.md

```diff
@@ -287,3 +287,21 @@ The following is the result of searching "6.858 Computer Systems Security site:g
 ### Information System Engineering
 > SE348 - 信息系统工程
 * [tx19980520-2019](https://github.com/tx19980520/ISEs)
+
+### Software Engineering Practice (SEP)
+
+> SE127 - 软件基础实践
+
+* [shenhliu-2018](https://github.com/shenhliu/SJTU-SE127-SEP)
+
+### Machine Learning
+
+> SE125 - 机器学习
+
+* [shenhliu-2018](https://github.com/shenhliu/SJTU-SE125-machine-learning)
+
+### Algorithm design and Implementation
```

Comment:
Please capitalize the title (i.e., Algorithm Design and Implementation).

Reply:
I have capitalized the title. Please check the request.

Reply:
Please check it again 😂